BoostTestAdapter/BoostTestAdapter.csproj

```diff
@@ -60,8 +60,8 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <FilesToLocalize Include="$(OutDir)\BoostTestAdapter.dll">
-      <TranslationFile>$(MSBuildThisFileDirectory)..\loc\lcl\{Lang}\BoostTestAdapter.dll.lcl</TranslationFile>
+    <FilesToLocalize Include="$(OutDir)\BoostTestAdapter.TestAdapter.dll">
+      <TranslationFile>$(MSBuildThisFileDirectory)..\loc\lcl\{Lang}\BoostTestAdapter.TestAdapter.dll.lcl</TranslationFile>

```

Comment:
Sorry to not be more specific initially, but changing the name of the lcl file in the TranslationFile property will break localization until we get another localization merge. If you can change this file name back, it will pull the old translations in still and I can make sure to update it once we have the localizations regenerated for the project.